static/_headers

```diff
@@ -14,7 +14,7 @@
 
 
 # Allow ACAO for well-known client
```

Comment:
We can go for `*` but update this comment too then.

Reply:
Sorry for the delay, I've updated the comment now.